### PR TITLE
Milestone 1: Hello Microverse

### PR DESCRIPTION
**Pull request by abelagza**
# This is the **Hello Microverse** project.

## in this Milestone I:

- Created a repository and cloned it to my desktop
- Created branch add_new_title
- Added .gitignore file
- Set Up linters for HTML and CSS 
- Created HTML and CSS files.
-

### CODE REVIEW

File: .github/linters.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+name: Linters
+
```
Comment (Amrhub) [inline .github/linters.yml]: - [ ] That's the correct workflow but you have it inside `.github/linters.yml` it should be `.github/workflows/linters.yml` if you are wondering why tests don't work. Other than that just good job.
- [ ] Let's make sure next time we have a descriptive and meaningful commit message your messages are accurate but more generic we need to be using commit messages that look like that `added markup to index page` a little bit meaningful and descriptive accurate :shipit:

**Review by Amrhub — CHANGES_REQUESTED** [review]
Hi @abelagza,  👋🏻 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!
These changes aim to add some awesomeness to your awesome project and code skills.

## To Highlight ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 


## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .github/linters.yml (hunk 1 above)
Comment (abelagza) [reply .github/linters.yml]: Thanks for the feedback. 
 - I created the subfolder and moved the .yml file inside it.
 - I also tried giving those last 2 commits a more explicit name.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style.css">
+    <title>Hello Microverse</title>
+</head>
+<body>
+    <h1>Hello Microverse</h1>
```
Comment (Hamzaoutdoors) [inline index.html:11]: - [OPTIONAL] This requested change is optional as your first reviewer didn't mention it, Kindly, try to wrap this `h1` tag inside a `header` tag for more semantic reasons ^^

**Review by Hamzaoutdoors — APPROVED** [review]
Hi @abelagza, 

## Congratulations! 🎉 Project Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚀

To Highlight:

- [x] The Pull Request has a proper title and description. ✔️
- [x] You have added a descriptive Readme file. Good Job. ✔️
- [x] There is no linter error is present. ✔️
- [x] Project has met all the technical aspects you needed for this project! ✔️ 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
**N.A**

_Cheers and Happy coding and Keep rocking!_ 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**